locale.html

```diff
@@ -490,8 +490,9 @@ <h1><ins>get Intl.Locale.prototype.timeZones</ins></h1>
       <emu-alg>
         1. <ins>Let _loc_ be the *this* value.</ins>
         1. <ins>Perform ? RequireInternalSlot(_loc_, [[InitializedLocale]]).</ins>
+        1. <ins>Let _locale_ be _loc_.[[Locale]].</ins>
```

Comment:
Extra line?

Reply:
No, this is needed in this timeZones function because the next line need it.